_includes/manuals/nightly/3.5.3_database_setup.md

```diff
@@ -1,46 +1,10 @@
 
-Foreman is a rails application. Therefore, anything that is supported under RAILS (sqlite, mysql, postgresql, ...) can be used. See <a href="manuals/{{page.version}}/index.html#3.3InstallFromPackages">3.3 Install From Packages</a> for a list of packages for connecting foreman to the databse of your choice. At this time, Oracle DB is known to not work. Patches are welcome!
+Foreman is a Rails application. While Rails supports different databases, Foreman supports only PostgreSQL for production deployments.
 
 The database configuration file can be found at:
 
 <pre>/etc/foreman/database.yml</pre>
 
-#### SQLite (default)
-
-<div class="alert alert-info">When using SQLite, you can install the
-foreman-sqlite package. See <a href="manuals/{{page.version}}/index.html#3.3InstallFromPackages">3.3 Install From Packages</a>.</div>
-
-By default, the database will be created in the db subdirectory.
-
-{% highlight ruby %}
-production:
-  adapter: sqlite3
-  database: db/production.sqlite3
-  pool: 5
-  timeout: 5000
-{% endhighlight %}
-
-#### MySQL
-
-<div class="alert alert-info">When using MySQL, you can install the
-foreman-mysql2 package. See <a href="manuals/{{page.version}}/index.html#3.3InstallFromPackages">3.3 Install From Packages</a>.</div>
-
-Edit your config/database.yml and modify:
-{% highlight ruby %}
-production:
-  adapter: mysql2
-  database: foreman
-  username: foreman
-  password: password
-  host: localhost
-  socket: "/var/run/mysqld/mysqld.sock"
-{% endhighlight %}
-
-Afterwards you would need to re-populate your database, simply execute:
-
-    foreman-rake db:migrate
-    foreman-rake db:seed
-
 #### PostgreSQL
```

Comment:
Is this header still needed if just before we state it's the only supported production deployment?

Reply:
good point, removed and reworded the next sentence.